.circleci/config.yml

```diff
@@ -93,14 +93,20 @@ jobs:
             equal: ['linux', <<parameters.os>>]
           steps:
             - run: yarn add $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME#$CIRCLE_SHA1
+      - run:
+          # STL could also have an SDR inside it.  Take it out to prevent conflicts
+          name: remove sdr from source-tracking
+          command: |
+            npm install shx -g
+            shx rm -rf node_modules/@salesforce/source-deploy-retrieve
+          working_directory: node_modules/@salesforce/source-tracking
```

Comment:
Is this a common thing to do for circular deps? 
Does this not mean that for the tests source-tracking would be using the current version of SDR, but after deploy it'd be using whatever version it specifies in it's package.json?

Reply:
they're not circular, just transitive.  plugin-source depends on SDR and STL and STL depends on SDR.

When we build the CLI, we bump both SDR and STL via yarn resolutions to their latest version.

This tests the changes on this branch everywhere that SDR is used.  Imagine a bug in SDR that causes problems for STL, but you don't catch it because the nuts are using an STL with a different (bug-free) SDR.  Wouldn't want to miss that!